experimental/transformers-to-ctranslate/args.py

```diff
@@ -13,23 +13,6 @@ def make_parser():
             "or path to a directory containing the pretrained model."
         ),
     )
-    parser.add_argument(
-        "--activation_scales",
-        help=(
-            "Path to the pre-computed activation scales. Models may "
-            "use them to rescale some weights to smooth the intermediate activations "
-            "and improve the quantization accuracy. See "
-            "https://github.com/mit-han-lab/smoothquant."
-        ),
-    )
-    parser.add_argument(
-        "--copy_files",
-        nargs="+",
-        help=(
-            "List of filenames to copy from the Hugging Face model to the converted "
-            "model directory."
-        ),
-    )
     parser.add_argument(
```

Comment:
should this be given a reasonble default like `main`? or what if `None` is passed? will snapshot download api determine it automatically?

Reply:
yes snapshot download api automatically finds the newest.

https://huggingface.co/docs/huggingface_hub/guides/download#download-an-entire-repository

`downloads the latest revision by default. If you want a specific repository revision, use the revision parameter...`

Reply:
in that case, let's remove this arg and pass `None` for now.